dts/arm/st/wl/stm32wl.dtsi

```diff
@@ -219,6 +264,108 @@
 			status = "disabled";
 			label = "SPI_2";
 		};
+
+		subghzspi: spi@58010000 {
+			compatible = "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x58010000 0x400>;
+			interrupts = <44 5>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB3 0x00000001>;
+			status = "disabled";
+			label = "SUBGHZSPI";
+		};
+
+		adc1: adc@40012400 {
+			compatible = "st,stm32-adc";
+			reg = <0x40012400 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000200>;
+			interrupts = <18 0>;
+			status = "disabled";
+			label = "ADC_1";
+			#io-channel-cells = <1>;
+		};
+
+		dac1: dac@40007400 {
+			compatible = "st,stm32-dac";
+			reg = <0x40007400 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x20000000>;
+			status = "disabled";
+			label = "DAC_1";
+			#io-channel-cells = <1>;
+		};
+
+		timers1: timers@40012c00 {
+			compatible = "st,stm32-timers";
+			reg = <0x40012c00 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000800>;
+			interrupts = <23 0>, <24 0>, <25 0>, <26 0>;
+			interrupt-names = "brk", "up", "trgcom", "cc";
+			status = "disabled";
+			label = "TIMERS_1";
+
+			pwm {
+				compatible = "st,stm32-pwm";
+				status = "disabled";
+				st,prescaler = <10000>;
```

Comment:
As other Series will most likely switch to have a prescaler of 0 per default(issue: #31290),
 i'd prefer to use a prescaler of 0 for the newly introduced timers.

Reply:
Cool, did not realize the difference, makes sense, changed them all to 0.